doc/memos/rdm0.md

```diff
@@ -0,0 +1,146 @@
+- RIOT Maintainers Community (RIOT-MC)  
+- Developer Memo: 0  
```

Comment:
It's not confusing, just adds another point of failure in the aforementioned process (both contributor and reviewer think about renaming (because in most cases git most likely actually warns about a conflict because there is already a file named `rdm42.md`), but forget to change the number in the document).